.devcontainer/Dockerfile.dev

```
@@ -27,14 +25,21 @@ RUN pip3 install pre-commit
 ####################################################
 # Install django
 ####################################################
-RUN pip3 install django==4.0 djangorestframework django-cors-headers
+RUN pip3 install \
+        django==4.0.1\
+        djangorestframework==3.13.1 \
+        django-cors-headers==3.12.0 \
+        django-oauth-toolkit==2.0.0\
+        psycopg2-binary==2.9.3
```

tzdata is listed in requirements.txt but not installed and the django admin console crashes without it 
```suggestion
        psycopg2-binary==2.9.3 \
        tzdata==2022.1
```